src/app/resources/view-resources/resources-view.component.ts

```diff
@@ -17,7 +17,62 @@ import { Subject } from 'rxjs/Subject';
       height: 80vh;
       border: none;
     }
-  ` ]
+    .resRating {
+      display: grid;
+      max-width: 230px;
+      margin-left: auto;
+    }
+    .rating {
+      text-align: center;
+    }
+    $gray: #aaa;
+
+    body{

```

Comment:
This style affects the entire app.  Do not add styles to app wide tags/component like `body` in a specific component style.